examples/lambda-functions/put-item.ts

```diff
@@ -56,7 +56,7 @@ export const handler = async (event: APIGatewayProxyEvent, context: Context): Pr
   });
 
   // Request a sample random uuid from a webservice
```

Comment:
Change the `logger.info('Incoming event` to `logger.info('Lambda invocation event` to align with what is logged by the middleware/decorator